rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/SystemProperty.java

```diff
@@ -87,6 +93,8 @@
     public static final String SERVICES_PACKAGE_PREFIX_DEFAULT = "uk.gov.gchq.gaffer.rest";
     public static final String BASE_PATH_DEFAULT = "rest";
     public static final String CORE_VERSION = "2.0.0";
+    public static final String GAFFER_VERSION_DEFAULT = getVersion(GAFFER_VERSION);
+    public static final String KORYPHE_VERSION_DEFAULT = getVersion(KORYPHE_VERSION);
```

Comment:
Perhaps you could optimise this as it has to load the version.properties file twice, once to get the Gaffer version, then again to get the Koryphe version.